### PR TITLE
docs: remove compound executor and graph document

### PR DESCRIPTION
**Pull request by Roshanjossey**
Only from toc

### CODE REVIEW

**Comment by hanxiao** [comment]
windows was already moved to install
image

File: docs/advanced/experimental/index.md [hunk 1]
```diff
@@ -17,6 +17,5 @@ That said, have fun!
 
 kubernetes
 indexers
-compound-executor
-graph-document
+windows
```
Comment (hanxiao) [inline docs/advanced/experimental/index.md:20]: no need

**Review by hanxiao — CHANGES_REQUESTED** [review]
keep the md, only show remove  `compound-executor, graph-document` in TOC would be enough

**Comment by Roshanjossey** [comment]
Updated to address review comments

**Pull request merged**